### PR TITLE
Various Floofstation Ports.

### PR DESCRIPTION
**Pull request by DangerRevolution**
# Description

Various minor yaml and QOL ports from Floofstation.

Changelog entries go in order of author. I'm considering just copying over the changelog at some point.

# Changelog

:cl: AeraAuling, ShatteredSwords and Cynical24
- add: Added the Explorer's Labcoat and a Hairflower into Loadouts.
- fix: Fixed Epistemics being unable to take Survival Boxes
- fix: Fixed the NearSighted trait making you unable to take glasses in your loadout
- add: Added a variety of Cowboy Hats into loadouts.

### CODE REVIEW

File: Resources/Prototypes/Loadouts/head.yml [hunk 1]
```diff
@@ -107,6 +155,14 @@
   items:
     - ClothingHeadHatBellhop
 
+- type: loadout
+  id: LoadoutHeadFlower
+  category: Head
+  cost: 1
+  exclusive: true
+  items:
+    - ClothingHeadHatHairflower
```
Comment (VMSolidus) [inline Resources/Prototypes/Loadouts/head.yml:164]:
```suggestion
```

Hairflower got removed in a recent update, but now you can just wear a poppy on your head that has exactly the same appearance as before.

**Review by VMSolidus — DISMISSED** [review]

File: Resources/Prototypes/Loadouts/head.yml (hunk 1 above)
Comment (DangerRevolution) [reply Resources/Prototypes/Loadouts/head.yml:164]: Why? Can we re-add that?

Comment (DangerRevolution) [reply Resources/Prototypes/Loadouts/head.yml]: resolved

**Comment by DangerRevolution** [comment]
Didn't realise these glasses don't exist lol, will fix and reopen soon.

**Review by VMSolidus — APPROVED** [review]

**Review by OldDanceJacket — APPROVED** [review]

**Pull request merged**